src/flb_input.c

```diff
@@ -1670,6 +1679,24 @@ int flb_input_test_pause_resume(struct flb_input_instance *ins, int sleep_second
     return 0;
 }
 
+static void flb_input_any_overlimit_paused(struct flb_input_instance *ins)
+{
+    if (ins->cmt_any_overlimit != NULL) {
+        /* cmetrics */
+        cmt_gauge_set(ins->cmt_any_overlimit, cfl_time_now(), 1,
+                      1, (char *[]) {ins->name});
+    }
+}
+
+static void flb_input_any_overlimit_resumed(struct flb_input_instance *ins)
+{
+    if (ins->cmt_any_overlimit != NULL) {
+        /* cmetrics */
+        cmt_gauge_set(ins->cmt_any_overlimit, cfl_time_now(), 0,
+                      1, (char *[]) {ins->name});
```

Comment:
when using the instance name as a label, there are cases where the user sets an alias for it, so it's recommended to use the wrapper [flb_input_name()](https://github.com/fluent/fluent-bit/blob/master/src/flb_input.c#L648) to get the proper name to use.

Reply:
Oh, I had overlooked that. Fixed in https://github.com/fluent/fluent-bit/pull/8044/commits/1326d01ad140a9a77c924b1584e71a1d4532246e.